libnavigation-core/api/current.txt

```diff
@@ -760,12 +760,13 @@ package com.mapbox.navigation.core.routerefresh {
 
   @com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI public final class RouteRefreshExtra {
     field public static final com.mapbox.navigation.core.routerefresh.RouteRefreshExtra INSTANCE;
+    field public static final String REFRESH_STATE_CANCELED = "CANCELED";
```

Comment:
https://github.com/mapbox/mapbox-navigation-android/pull/6345/files#diff-76ce9cbef0d51b7f1bac812adf4f28cba6e43003aadcfd4c4d78f35ed5ef5a62 didn’t include `REFRESH_STATE_CANCELED`. As `RouteRefreshExtra` was a new class check didn't fail, I believe https://github.com/mapbox/mapbox-navigation-android/issues/6257 would have caught it.

cc @RingerJK

Reply:
great catch. seems metalava is not working properly

Reply:
ref `#NAVAND-664`